_contributors-app/sudonitesh.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Nitesh Tosniwal
+description: I am an aspiring full stack web developer.
+picUrl: https://avatars2.githubusercontent.com/u/35661486?s=460&v=4
+skills: |
+  - **Languages:** Javascript, Python, HTML/CSS, C/C++
+  - **Frameworks:** NodeJS, ExpressJS, Jquery
+  - **Databae:** MongoDB, MySQL
+  - **Tools:** git, Github, TravisCI
+
+githubUsername: sudonitesh
+speciality: |
+
+  - Developing Responsive Full Stack Web Apps
+
+  - Open Source
+
+github_url: https://github.com/sudonitesh
+linkedIn_url: https://www.linkedin.com/in/sudonitesh/
+twitter_url: https://twitter.com
```

Comment:
@sudonitesh give your correct Twitter url.

Reply:
Have you seen me on twitter?!! I don't have one..